<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigInicial</class>
 <widget class="QDialog" name="frmConfigInicial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuración Inicial del GR-lida</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gr-lida.qrc">
    <normaloff>:/img32/gr-lida.png</normaloff>:/img32/gr-lida.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lb_imagenlogo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>72</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>72</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/images/gr-lida_logo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="lb_Confini_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Seleccionar idioma:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QGroupBox" name="chkScummVMDisp">
       <property name="title">
        <string>Tengo instalado el emulador ScummVM</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_Confini_4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Dirección del ScummVM:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSvmPath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSvmPath">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSvmPath_find">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/zoom.png</normaloff>:/img16/zoom.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSvmPath_clear">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QGroupBox" name="chkDOSBoxDisp">
       <property name="title">
        <string>Tengo instalado el emulador DOSBox</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_Confini_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Dirección del DOSBox:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtDbxPath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDbxPath">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDbxPath_find">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/zoom.png</normaloff>:/img16/zoom.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDbxPath_clear">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="cbxIdioma">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="lb_Confini_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>54</height>
        </size>
       </property>
       <property name="text">
        <string>Esta es la primera vez que lo abres o todaví­a no lo has configurado y para poder utilizar el programa correctamente debes configurarlo primero con estas sencillas opciones</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="lb_Confini_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Bienvenidos a GR-lida</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="chkShowNext">
       <property name="text">
        <string>Mostrar la próxima vez que inicie</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QCheckBox" name="chkVDMSoundDisp">
       <property name="text">
        <string>Tengo instalado el emulador VDMSound</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxIdioma</tabstop>
  <tabstop>chkDOSBoxDisp</tabstop>
  <tabstop>btnDbxPath_find</tabstop>
  <tabstop>btnDbxPath</tabstop>
  <tabstop>txtDbxPath</tabstop>
  <tabstop>btnDbxPath_clear</tabstop>
  <tabstop>chkScummVMDisp</tabstop>
  <tabstop>btnSvmPath_find</tabstop>
  <tabstop>btnSvmPath</tabstop>
  <tabstop>txtSvmPath</tabstop>
  <tabstop>btnSvmPath_clear</tabstop>
  <tabstop>chkVDMSoundDisp</tabstop>
  <tabstop>chkShowNext</tabstop>
  <tabstop>btnOk</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
